lxd/containers.go

```diff
@@ -247,6 +248,21 @@ func containersRestart(s *state.State) error {
 	return nil
 }
 
+func vmMonitor(s *state.State) error {
```

Comment:
@stgraber perhaps we could move this into a `vms.go` file as being in `containers.go` doesn't seem appropriate.

Reply:
I was expecting us to eventually rename this to instances.go at which point things will make more sense